### PR TITLE
Changes to README for JOSS review

### PR DESCRIPTION
**Pull request by tomgrylls**
Two sections have been added to README.md in response to #137:

1. An overview section to illustrate the statement of need for uDALES in its documentation.
2. A Publications and validation section to demonstrate that uDALES is "capable of explicitly representing buildings and of modelling the full range of urban processes" in its documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,27 @@ This is the development repository for the uDALES (urban Dutch Atmospheric Large
...
+
+The turbulent nature of the urban flow field and the inherent heterogeneity and wide range of scales associated with the urban environment result in a complex modelling problem. Large-eddy simulation (LES) is an approach to turbulence modelling used in computational fluid dynamics to simulate turbulent flows over a wide range of spatial and temporal scales. LES is one of the most promising tools to model the interactions typical of urban areas due to its ability to resolve the urban flow field at resolutions of _O_(1 m, 0.1 s), over spatial domains of _O_(100 m), and time periods of _O_(1 h). Although there are many scalable LES models for atmospheric flows, to our knowledge, only few are capable of explicitly representing buildings and of modelling the full range of urban processes (e.g. PALM-4U; Resler et al. 2017; Maronga et al. 2020; or OpenFoam; Weller et al. 1998)..
```
Comment (dmey) [inline README.md:14]:
```suggestion
The turbulent nature of the urban flow field and the inherent heterogeneity and wide range of scales associated with the urban environment result in a complex modelling problem. Large-eddy simulation (LES) is an approach to turbulence modelling used in computational fluid dynamics to simulate turbulent flows over a wide range of spatial and temporal scales. LES is one of the most promising tools to model the interactions typical of urban areas due to its ability to resolve the urban flow field at resolutions of _O_(1 m, 0.1 s), over spatial domains of _O_(100 m), and time periods of _O_(1 h). Although there are many scalable LES models for atmospheric flows, to our knowledge, only few are capable of explicitly representing buildings and of modelling the full range of urban processes (e.g. PALM-4U; Resler et al. 2017; Maronga et al. 2020; or OpenFoam; Weller et al. 1998).
```

**Review by dmey — CHANGES_REQUESTED** [review]

**Comment by dmey** [comment]
@tomgrylls was thinking of branching this off the joss-paper branch but I think it's fine to merge it into master for this one. Only made a minor comment re type rest looks good with me. 👍

File: README.md (hunk 1 above)
Comment (bss116) [inline README.md:14]: I would maybe add a sentence like "uDALES is a large-eddy simulation model for urban environments" before saying: Large-eddy simulation is an approach to turbulence modelling ... and maybe remove the last sentence where we reference other models, as this is not really important for the code repository description.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,27 @@ This is the development repository for the uDALES (urban Dutch Atmospheric Large
...
+| Urban climate (radiation, green roofs and walls, trees etc.) | Suter (2018); Suter et al. (2021); Grylls & van Reeuwijk (2021)                 |
+| Pollution dispersion/ urban air quality                      | Grylls et al. (2019);  Grylls (2020)                                            |
+| Buoyancy/ convective and stable conditions                   | Suter (2018); Grylls et al. (2019); Grylls (2020); Grylls & van Reeuwijk (2021) |
+
+uDALES has been validated against both field and experimental studies and evaluated against other numerical models. The dynamic core, DALES, has been used to study atmospheric processes for over 30 years, being validated extensively in the process and used in several atmospheric intercomparison studies (Nieuwstadt and Brost 1986; Heus et al. 2010). The use of the immersed boundary method to model obstacles was validated by Tomas et al. (2016). The wall functions and three-dimensional surface energy balance model was validated by Suter (2018). The ability to model pollutuon dispersion in both idealised and realistic morphologies was been validated by Grylls et al. (2019) and Grylls (2020).
```
Comment (bss116) [inline README.md:29]: maybe a direct link to the DALES repository? The dynamic core, [DALES](https://github.com/dalesteam/dales),

File: README.md [hunk 3]
```diff
@@ -24,3 +45,23 @@ General DALES copyright applies for any files part of the original DALES distrib
...
+
+Grylls, T., & van Reeuwijk, M. (2021). Tree model with drag, transpiration, shading and deposition: Identification of cooling regimes and large-eddy simulation. Agricultural and Forest Meteorology, 298-299, 108288. https://doi.org/https://doi.org/10.1016/j.agrformet.2020.108288
+
+Oke, T. R., Mills, G., Christen, A., & Voogt, J. A. (2017). Urban climates. Cambridge University Press. https://doi.org/10.1017/9781139016476
+
+Suter, I. (2018). Simulating the impact of blue-green infrastructure on the microclimate of urban areas. https://doi.org/10.25560/78715
+
+Suter, I., Grylls, T., Sützl, B. S., & van Reeuwijk, M. (2021). uDALES 1.0.0: A large-eddy simulation model for urban environments. In Geoscientific Model Development. Copernicus GmbH.
+
+Sützl, B. S., Rooney, G. G., & van Reeuwijk, M. (2020). Drag distribution in idealized heterogeneous urban environments. Boundary-Layer Meteorology. https://doi.org/10.1007/s10546-020-00567-0
```
Comment (bss116) [inline README.md:65]: BLM has now printed the paper, so the reference needs to change to this:
Sützl, B. S., Rooney, G. G., & van Reeuwijk, M. (2021). Drag distribution in idealized heterogeneous urban environments. Boundary-Layer Meteorology, 178, 225-248. https://doi.org/10.1007/s10546-020-00567-0
Sorry I know this needs to change everywhere now.

**Review by bss116 — APPROVED** [review]
A few small suggestions from my side.

File: README.md [hunk 4]
```diff
@@ -7,6 +7,27 @@ This is the development repository for the uDALES (urban Dutch Atmospheric Large
...
+
+uDALES is capable of modelling urban environments at the microclimate scale including wet thermodynamics, idealized and complex morphologies, three-dimensional urban surface energy balance and real-time pollution dispersion.
+
+## Publications and validation
+
+To-date, uDALES has been used to study a number of research questions:
+
+| Research application                                         | Reference                                                                       |
+| ------------------------------------------------------------ | ------------------------------------------------------------------------------- |
+| Urban boundary layers/ boundary-layer meteorology            | Grylls et al. (2019); Sutzl et al. (2020)                                       |
```
Comment (bss116) [inline README.md:24]: Also I'd like to have my dots please :) Sützl et al. (2021)

**Review by bss116 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tomgrylls) [reply README.md]: Amended

Comment (tomgrylls) [reply README.md]: I have added the suggested first sentence.

I think we want to keep in the last sentence of that paragraph as there is a need to show a clear statement of need in the documentation (#137) and I think this point is key to understanding what uDALES offers to the field.

File: README.md (hunk 2 above)
Comment (tomgrylls) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (tomgrylls) [reply README.md]: I have I have updated and I will also update in the JOSS paper.

File: README.md (hunk 4 above)
Comment (tomgrylls) [reply README.md]: Of course!

**Review by dmey — APPROVED** [review]

**Review by bss116 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (bss116) [reply README.md]: Thanks!

**Pull request merged**